backend/src/router.py

```diff
@@ -23,16 +24,12 @@ def build_best_next_step_prompt(task, scratchpad):
 def build_plan(task, scratchpad):
     best_next_step_prompt = build_best_next_step_prompt(task, scratchpad)
 
-    # Call model to choose agent
-    logging.info("#####  ~  Calling LLM for next best step  ~  #####")
-
-    logging.info("Scratchpad so far:")
-    logging.info(scratchpad)
+    logger.info("#####  ~  Calling LLM for next best step  ~  #####")
+    logger.info(f"USER - Scratchpad so far: {scratchpad}")
```

Comment:
Can I ask what 'USER' represents here and why you're using it at times and not others?

Reply:
My understanding is that this is to represent what we intend to pass to the frontend. Correct me if i'm wrong @gaganahluwalia

Reply:
@bheyman-scottlogic  is right, USER is to represent that we intend to pass this one to Frontend in the future.